sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -297,5 +274,69 @@ public virtual async Task CancelAsync(CancellationToken cancellationToken = defa
                 throw;
             }
         }
+
+        /// <summary>
+        /// Gets the final result of the long-running operation in an asynchronous way.
+        /// </summary>
+        /// <remarks>
+        /// Operation must complete successfully (HasValue is true) for it to provide values.
+        /// </remarks>
+        public override AsyncPageable<AnalyzeHealthcareEntitiesResultCollection> GetValuesAsync()
+        {
+            if (!HasCompleted)
+                throw new InvalidOperationException("The operation has not completed yet.");
+            if (HasCompleted && !HasValue)
```

Comment:
Because of the previous statement, I think this can be simplified to:
```suggestion
            if (!HasValue)
```

Reply:
updated